orchestration/templates/ingest-file.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ingest-file
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: url
+          - name: dataset-id
+          - name: profile-id
+          - name: gcs-bucket
+          - name: gcs-file-path
+          - name: target-path
+          - name: timeout
+      dag:
+        tasks:
+          # 1) Make sure that something doesn't already exist at the target path
+          - name: check-existing-file
+            template: check-existing-file
+            arguments:
+              parameters:
+                - name: api-url
+                  value: '{{ "{{inputs.parameters.url}}/api/v1" }}'
```

Comment:
I still wouldn't do the `/api/v1`-appending at the Argo layer. The scripts should specify the full endpoints they depend on.